waspc/src/Wasp/Generator/Shared.hs

```diff
@@ -0,0 +1,31 @@
+module Wasp.Generator.Shared
+  ( mkTmplFdWithDst,
+    SharedTemplatesDir,
+  )
+where
+
+import qualified Data.Aeson as Aeson
+import StrongPath (Dir, File', Path', Rel, reldir, (</>))
+import Wasp.Generator.Common (ProjectRootDir)
+import Wasp.Generator.FileDraft (FileDraft, createTemplateFileDraft)
+import Wasp.Generator.Templates (TemplatesDir)
+
+data SharedTemplatesDir
+
+mkTmplFdWithDst :: Path' (Rel SharedTemplatesDir) File' -> Path' (Rel ProjectRootDir) File' -> FileDraft
+mkTmplFdWithDst src dst = mkTmplFdWithDstAndData src dst Nothing
+
+-- | Path in templates directory where shared templates reside.
```

Comment:
I also vote for dropping this comment (I'm aware you probably copied it over from existing generators). Still, I think the information given by the StrongPath type is plenty :)